tap-snapshots/test/lib/commands/view.js.test.cjs

```diff
@@ -82,7 +82,7 @@ dist
 dist-tags:
 [1m[32mlatest[39m[22m: 1.0.0
 
-published [33myesterday[39m
```

Comment:
not sure why this changed ended up in your tests but it's making CI fail. Likely just need to revert this one to match`release-next`, and it's something for us to look into.